### PR TITLE
engine: add release notes from 19.03.6 and 19.03.7

### PR DESCRIPTION
**Pull request by tiborvass**
fixes https://github.com/docker/docker.github.io/issues/10407

Signed-off-by: Tibor Vass <tibor@docker.com>

Ping @usha-mandya @thaJeztah

### CODE REVIEW

**Comment by tiborvass** [comment]
Unsure why it's failing. There's only a warning that says:

> npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.

But if it's a warning it shouldn't cause a failure. 🤷‍♂

**Comment by thaJeztah** [comment]
I think there were issues with passing the account info

```
Usage: <netlify_auth_token> <netlify_account_slug> <site_name> <directory>
+ usage
+ echo 'Usage: <netlify_auth_token> <netlify_account_slug> <site_name> <directory>'
+ exit 1
```

File: engine/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,54 @@ for Docker Engine - Community.
...
+2020-03-03
+
+### Builder
+
+- builder-next: Fix deadlock issues in corner cases. [moby/moby#40557](https://github.com/moby/moby/pull/40557)
+
+### Runtime
+
+* overlay: remove modprobe execs. [moby/moby#40462](https://github.com/moby/moby/pull/40462)
+* selinux: better error messages when setting file labels [moby/moby#40547](https://github.com/moby/moby/pull/40547)
```
Comment (usha-mandya) [inline engine/release-notes.md]: Suggest updating to `display better error messages when setting file labels.`

**Comment by usha-mandya** [comment]
@thaJeztah Wondering if it's a good idea to unhook this build check until we fix the Netlify account issue?

**Comment by thaJeztah** [comment]
> Wondering if it's a good idea to unhook this build check until we fix the Netlify account issue?

Yes, if the "deploy to metlife" step is non-functional, we may as well remove it for the time being

**Comment by usha-mandya** [comment]
OK, thanks @thaJeztah.  I've updated the branch protection rules. As soon as @tiborvass makes the minor update, we should be able to approve and push it.

**Comment by tiborvass** [comment]
Updated

**Pull request merged**